extensions/database/pom.xml

```diff
@@ -135,7 +135,7 @@
     <dependency>
       <groupId>com.google.http-client</groupId>
       <artifactId>google-http-client-jackson2</artifactId>
-      <version>1.24.1</version>
+      <version>1.20.0</version>
```

Comment:
Was there some problem with the newer jackson2 client ?  Can you explain the "unifying" reason ?

Reply:
@thadguidry the database extension uses this older version of Jackson - I migrated that to Jackson2 when writing the pom.xml file but it turns out that this introduces some runtime exceptions. I am keeping the dependencies as they are now for this PR to minimize the risk of introducing - it can be migrated later, in a different PR.